aws/utility.sh

```diff
@@ -93,14 +93,22 @@ function trace() {
   message "${LOG_LEVEL_TRACE}" "$@"
 }
 
-function info() {
+function information() {
   message "${LOG_LEVEL_INFORMATION}" "$@"
 }
 
+function info() {
+  information "$@"
+}
+
 function warning() {
   message "${LOG_LEVEL_WARNING}" "$@"
 }
 
+function warn() {
```

Comment:
Do we really need this? feels like we should just find and replace where we are using the wrong ones